CONTRIBUTING.md

```diff
@@ -0,0 +1,36 @@
+### Contributing
+
+- When contributing to this repository, please first discuss the change you wish to make via issue, email, or any other method with the owners of this repository before making a change.
+
+### We Develop with Github
```

Comment:
Not really necessary to have this section since the repository is hosted on GitHub anyways.

Reply:
Ok let me remove this